src/Workspaces/CSharp/Portable/Extensions/SyntaxTreeExtensions.cs

```diff
@@ -574,6 +567,39 @@ private static bool AtEndOfIncompleteStringOrCharLiteral(SyntaxToken token, int
             }
         }
 
+        private static bool IsSelected(TextSpan textSpan, MemberDeclarationSyntax member, bool allowPartialSelection)
```

Comment:
Can this be a local function? Otherwise, the name should indicate it is only for fields and properties.

Reply:
I renamed it and also made it a local function in CS.